common/predictive-text/unit_tests/test.sh

```diff
@@ -9,114 +9,168 @@ THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BA
 . "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
 SCRIPT_ROOT="$(dirname "$THIS_SCRIPT")"
 
-# A simple utility script to facilitate unit-testing for the LM Layer.
-# It's rigged to be callable by NPM to facilitate testing during development when in other folders.
-
-display_usage ( ) {
-  echo "test.sh [ -? | -h | -help]"
-  echo "  -CI                    to perform continuous-integration friendly tests and reporting"
-  echo "  -headless              to disable the in-browser tests"
-  echo "  -integrated            to disable the 'headless' test suite"
-  echo "  -skip-package-install  (or -S) skips dependency updates"
-  echo "  -? | -h | -help        to display this help information"
-  echo ""
-  exit 0
-}
+# This script runs from its own folder
+cd "$SCRIPT_ROOT"
```

Comment:
```suggestion
cd "$THIS_SCRIPT_PATH"
```

Given this is defined in build-utils.sh, let's converge on that name.